pkgs/development/compilers/flutter/sdk-symlink.nix

```diff
@@ -6,6 +6,12 @@ let
       name = "${flutter.name}-sdk-links";
       paths = [ flutter flutter.sdk ];
 
+      # The final flutter binary is required to be a git repo.
```

Comment:
I don't want to be relying on `.git` directories from any input derivations if we can avoid it. If we ever switch to using Flutter from `fetchFromGitHub` (which is feasible now that we fetch our own artifacts and a Nix-built engine is a WIP), this will break.